nifgen test_error_message system test fails with NI-FGEN 2023Q3 due to a different error code being returned.

### Description of issue

The error behavior has changed for NI-FGEN 2023Q3 and because we expect a specific error code in test_error_message, the test fails. We should update the test to accept either error code.

Here is output from the failure:
```Python
AssertionError: assert -1074134964 == -1074134944
+ where -1074134964 = DriverError('-1074134964: IVI: The option string parameter contains an entry with an unknown option value.

Possible Values: 5402, 5404, 5406, 5412, 5413 (1CH), 5413 (2CH), 5421, 5422, 5423 (1CH), 5423 (2CH), 5433 (1CH), 5433 (2CH), 5441, 5442, 5450, 5451').code
```

--- a/src/nifgen/system_tests/test_system_nifgen.py
+++ b/src/nifgen/system_tests/test_system_nifgen.py
@@ -70,8 +70,15 @@ class SystemTests:
             with nifgen.Session('', '0', False, 'Simulate=1, DriverSetup=Model:invalid_model (2CH);BoardType:PXIe', **session_creation_kwargs):
                 assert False
         except nifgen.Error as e:
-            assert e.code == -1074134944
-            assert e.description.find('Insufficient location information or resource not present in the system.') != -1
+            # The returned error has changed over time, so accept multiple error codes, descriptions.
+            # Users should generally not look for specific error codes and should instead correct their code if they hit an error.
+            assert e.code in [-1074134944, -1074134964]
+            assert any(
+                [
+                    e.description.find('Insufficient location information or resource not present in the system.') != -1,
+                    e.description.find('The option string parameter contains an entry with an unknown option value.') != -1,
+                ]
+            )
 
     def test_get_error(self, session):
         try:

